File: engine/model/src/main/scala/tools/packet/cashshop/operation/ShowBoughtCharacterSlots.scala
package tools.packet.cashshop.operation

import net.opcodes.SendOpcode
import tools.packet.PacketInput

class ShowBoughtCharacterSlots(private var _slots: Short) extends PacketInput {
  def slots: Short = _slots

  override def opcode(): SendOpcode = SendOpcode.CASHSHOP_OPERATION
}

File: engine/engine-model/src/main/java/tools/packet/cashshop/operation/ShowBoughtCharacterSlots.java
package tools.packet.cashshop.operation;

import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record ShowBoughtCharacterSlots(Short slots) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.CASH_SHOP_OPERATION;
   }
}